### ISSUE TITLE
Error: No named parameter with the name 'size'. size: logicalSize. Flutter 3.2+ and Davinci 2.1.1 issue

### ISSUE THREAD

**Issue by saintjab** [Imgkl/davinci#36]
Using Davinci 2.1.1 and Flutter 3.2.0 and getting this error building my app for web.

`../../../.pub-cache/hosted/pub.dev/davinci-2.1.1/lib/core/davinci_capture.dart:88:11:
Error: No named parameter with the name 'size'.
          size: logicalSize,
          ^^^^
../../../Library/flutter/packages/flutter/lib/src/rendering/view.dart:28:9:
Info: Found this candidate, but the arguments don't match.
  const ViewConfiguration({
        ^^^^^^^^^^^^^^^^^
`
Anyone having this issue? It is now preventing me to release :(

**Comment by Imgkl** [comment]
@saintjab I'll check this and I'll get back to you. Can you share a small snippet of the code to reproduce this issue?

**Comment by saintjab** [comment]
Hey @Imgkl, am just using this
`DavinciCapture.offStage(davinciModel.widget, returnImageUint8List: true, context: davinciModel.buildContext,);`
to get the screenshot of a widget. 
The issue can be reproduced if you update to the latest flutter (3.2.2) and then generating a build with the 2.1.1 of Davinci in use.
`flutter build web --release`
You should get the error above.  It just started when I updated my flutter version. Downgrading back to lower version (say 3.19.4) seems to be fine. Cheers

**Comment by idouiri9806** [comment]
@saintjab @Imgkl  looks like the parameters for the `ViewConfiguration` has changed, remove `size` and use `logicalConstraints` something like this 
```ruby
  final RenderView renderView = RenderView(
        view: ui.PlatformDispatcher.instance.implicitView!,
        child: RenderPositionedBox(
          alignment: Alignment.center,
          child: repaintBoundary,
        ),
        configuration: ViewConfiguration(
          //size: logicalSize,
          logicalConstraints: BoxConstraints(
            minWidth: logicalSize.width,
            maxWidth: logicalSize.width,
            minHeight: logicalSize.height,
            maxHeight: logicalSize.height,
          ),
          devicePixelRatio: 1.0,
        ),
      );
```

**Comment by Imgkl** [comment]
Thanks @idouiri9806, I'll make this change and push a new version today.

**Comment by saintjab** [comment]
@Imgkl any update on when this expected to be live?

**Comment by vqh2602** [comment]
Looks like this package is no longer maintained. Probably have to find another solution

**Comment by vqh2602** [comment]
I found a similar package : https://pub.dev/packages/davincii

**Comment by Imgkl** [comment]
@vqh2602 This package is not discontinued or anything. I was just caught up with other stuffs. Now most of them are sorted out. I'll work on my OSS contirbutions.

**Comment by Imgkl** [comment]
Fixed on v2.5.1
